src/Chat/Bot/Vote.hs

```diff
@@ -52,16 +52,17 @@ createPoll question answers =
 --   map :: (a -> b) -> [a] -> [b]
 --
 castVote :: Poll -> String -> Poll
-castVote poll vote =
-  notImplemented "Vote.castVote" (castVoteAnswer poll vote)
+castVote (Poll question options) vote =
+  notImplemented "Vote.castVote" (castVoteAnswer (Poll question options) vote)
 
 -- | LEVEL: Medium
--- 
+--
 -- HINTS:
 --
 --   map :: (a -> b) -> [a] -> [b]
 --   unlines :: [String] -> String
+--   ++ :: [a] -> [a] -> [a]
 --
 pollRender :: Poll -> String
-pollRender poll =
-  notImplemented "Vote.pollRender" (pollRenderAnswer poll)
+pollRender (Poll question options) =
+  notImplemented "Vote.pollRender" (pollRenderAnswer (Poll question options))
```

Comment:
Just as aside you could do `p@(Poll question options)` and then refer to `p` as the `Poll`, but let's not go there. :)